site/en/posts/iwd-2022-shree-bose-piper.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Helping students fall in love with building technology - Shree Bose of Piper'
+metadesc: In honor of International Women's Day we interviewed Shree Bose, co-founder of Piper, about her quest to help people become passionate about creating technology.
+tags:
+  - success story
+  - web
```

Comment:
```suggestion
    - web
    - success story
```
Because this is going to fall under our "web" blog category we want web to come first